.github/workflows/python-package.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8]
```

Comment:
interestingly, our unit tests fail on python 3.9 (I think you can see the failed [build logs](https://github.com/Arun-Niranjan/pymdp/pull/3/checks?sha=9b38878948bb6ed7be926a5a4ec18747c06ed775) for the previous commit on this branch)

> E     File "/home/runner/work/pymdp/pymdp/test/test_control.py", line 28
> p = np.random.rand((*([no] + n_states)))
> E                             ^
> E   SyntaxError: can't use starred expression here

I think this should be a straightforward fix, but it should be done in a separate PR.

Reply:
Thanks, nice find. I need to look at Python 3.9 to see what the exact restrictions are, but  the 'official' numpy.random documentation recommends using `np.random.random_sample()` when passing in variable shape tuples (see https://numpy.org/devdocs/reference/random/generated/numpy.random.rand.html), so I will change those across the board when we're initializing random uniform matrices.